### PR TITLE
et: init at 2017-03-04

### PR DESCRIPTION
**Pull request by oxzi**
###### Motivation for this change
Adds the [et](https://github.com/geistesk/et)-software, which is a minimal libnotify-based (egg) timer for GNU/Linux.

###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/et/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  version = "2017-03-04";
+
+  src = fetchFromGitHub {
+    owner = "geistesk";
+    repo = "et";
+    rev = "151e9b6bc0d2d4f45bda8ced740ee99d0f2012f6";
+    sha256 = "1a1jdnzmal05k506bbvzlwsj2f3kql6l5xc1gdabm79y6vaf4r7s";
+  };
+
+  buildInputs = [ pkgconfig libnotify gdk_pixbuf ];
```
Comment (fpletz) [inline pkgs/applications/misc/et/default.nix]: `pkgconfig` should be in `nativeBuildInputs`.

File: pkgs/applications/misc/et/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    mkdir -p $out/bin
+    cp et $out/bin
+    cp et-status.sh $out/bin/et-status
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Minimal libnotify-based (egg) timer for GNU/Linux";
+    homepage = https://github.com/geistesk/et;
+    license = licenses.gpl3;
+    platforms = platforms.linux;
```
Comment (fpletz) [inline pkgs/applications/misc/et/default.nix]: I think we could go with `platforms.unix` here, right? The code looks portable enough. :)

**Comment by geistesk** [comment]
@fpletz: Thanks for your advice which I applied to my previous commit.

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**